vnext/Shared/Modules/WebSocketModule.cpp

```diff
@@ -75,166 +171,83 @@ std::vector<facebook::xplat::module::CxxModule::Method> WebSocketModule::getMeth
           }
         }
 
-        weak_ptr weakWs = this->GetOrCreateWebSocket(jsArgAsInt(args, 3), jsArgAsString(args, 0));
+        weak_ptr weakWs = GetOrCreateWebSocket(jsArgAsInt(args, 3), jsArgAsString(args, 0), weakState);
```

Comment:
>GetOrCreateWebSocket [](http://example.com/codeflow?start=26&length=20)

Hmm, so why not just do:
strongState = weakState.lock
if (!strongState) return
strongState->GetOrCreateWebSocket(...)
?

Reply:
This change removes `GetOrCreateWebSocket` as a member of `WebSocketModule` altogether and turns it into a static free function.

Same applies to `SendEvent`.

This was done to decouple the functionality that is not inherent to `CxxModule` and its lifetime.